### DISCUSSION TITLE
modifying tuplet and related in a bar

### DISCUSSION THREAD

**Discussion by blindfaced** [helge17/tuxguitar#226] [General]
you have a full bar, and in this bar eight notes triplets; for example. 
you want to change, to remove these triplets. 
but you can't because it would exceed the duration of the bar. 
so you'll have to change to sixteenth notes. then remove the triplets. and you'll end up with notes and silences galore.
look at Guitar pro, you can delete and change whatever you want; the duration is displayed and you can see if it's complete or overrunning, and if you switch to another bar, you'll see the wrong bar in red. simple and handy.
i do  think it could be a great step forward

**Comment by guiv42** [comment]
tracked by #579
